gapis/resolve/stats.go

```diff
@@ -90,17 +90,14 @@ func drawCallStats(ctx context.Context, capt *path.Capture, stats *service.Stats
 			if err != nil {
 				return err
 			}
-			// If the command has subcommands, ignore it (vkQueueSubmit or similar)
```

Comment:
Isn't this comment still relevant?

Reply:
No. The code below handles subcommands; the comment is just wrong.